lib/web_ui/test/text/line_breaker_test_helper.dart

```diff
@@ -0,0 +1,192 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.10
+
+const Map<String, String> _replacements = <String, String>{
```

Comment:
I'm not sure if there's a better way to encode these test modifications/replacements. Any ideas @yjbanov ?

Reply:
`Map<String, String>` is simple enough to grok. Maybe you could extract the keys and values into named constants? Not sure if it will help. Comments would probably add significant clarity though.